templates/elections.html

```diff
@@ -21,7 +21,7 @@ <h1 class="entity__name">{{ cycle }} Presidential election</h1>
           {% else %}
             <h1 class="entity__name">US {{ office|title }} election: {{ state|fmt_state_full }}</h1>
             {% if district %}
-              <h2>United States House District {{ district }}</h2>
+              <h2>United States House <span class="term" data-term="District">District</span> {{ district }}</h2>
```

Comment:
I think this shouldn't be a term because it's read as "District 2" not like "District: 2".